03-task-manager/starter/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "dotenv": "^8.2.0",
+    "dotenv": "^8.6.0",
```

Comment:
Did you run into any issues with having 2 package.json files?

Reply:
I had not, how would I go about resolving that?